modules/ROOT/pages/insight.adoc

```diff
@@ -23,208 +35,269 @@ Insight helps you answer questions about your integrated apps, such as:
 
-image::chtransactiondetails.png[CHTransactionDetails]
 
-The default settings of the events filter depend upon your user role as follows:
+== Diagnosing Problems and Replaying Transactions
 
-* If your user role is *Support*, the default events filter setting display only custom events.
-* If your user role is *Admin* or *Developer*, the default events filter setting displays custom events, endpoint events, and message processor events, but does not display flow events.
 
-== Data Persistence
+When Insight reports a failed transaction, you can diagnose the problem and fix the root issue. 
 
-Insight does not have a size limit and displays all transactions in your application for a period of 30-days. Events are kept for a period of 2-days.
+For Mule 3 applications and flows with inbound endpoints, you can use the Insight message replay feature to replay the transaction that failed.
+ 
+=== Transaction Failed Status
 
-[NOTE]
-Replay data for applications created before Oct 7, 2017 are stored in the default region: us-east-1. +
-New applications and applications whose region changed after Oct 7, 2017 use region-specific storage. Insights Replay Data is not preserved when switching regions. Verify that this doesn't violate any of your compliance requirements.
+For example, when a transaction fails because an application exceeds the limited number of API calls for a Salesforce account, Insight displays the status of the transaction as `Failed`. 
 
-[[diagnosing_problems]]
-== Diagnosing Problems and Replaying Transactions
+Click to expand the transaction, then examine the individual events within that transaction. The exception and the error message are listed next to the failed event, as well as an existing stack trace:
 
-Insight's message replay feature enables advanced error recovery. When you encounter a failed transaction (as displayed on Insight's dashboard), you can diagnose the problem, fix the root issue, then replay the transaction that failed. The following example demonstrates this functionality.
+image::chexcmsg.png[CHExcMsg]
 
-=== Transaction Failed
 
-For example, when a transaction fails because an application exceeds the limited number of API calls for a Salesforce account, Insight displays the status of the transaction as `Failed`. 
+=== Replay Transaction (Mule 3 apps only)
 
-Click to expand the transaction, then examine the individual events within that transaction. The exception and the error message are listed next to the failed event, as well as an existing stack trace.
+After investigating the logs and resolving the issue (for example, by purchasing more API capacity) that caused the transaction failure, you can replay the transaction.
+Replay is not supported for Mule 4 apps.
 
-image::chexcmsg.png[CHExcMsg]
+. Click the replay icon (circular arrow) to the right of the first event in the failed transaction to replay it:
```

Comment:
I just saw Valkyrie use a screen cap for an icon; suggest the same here.

Reply:
Also, if we show it in the capture directly following, we might not need to describe it here.